gui/public/sp_load_draws.py

```diff
@@ -0,0 +1,64 @@
+# Used in pyodideWorker for running analysis.py
+
+from typing import List
+import numpy as np
+
+
+class DrawsObject:
+    def __init__(self, sp_data):
```

Comment:
I'm sorry, I don't remember what `sp_data` is supposed to be or what assumptions we can make about it. Also, is there any type hinting here that would be appropriate?

Also it seems like we're only interested in the `sampling` member, should we narrow the parameter to just that?

Reply:
sp_data in Python corresponds to the spData on the javascript side. It's the way that the sampling draws are passed from js -> pyodide. Typing a dict is a lot easier in typescript than in Python. I didn't want to bring in pydantic or something heavy.

Reply:
I thought we might need to provide more information than just sampling in the future.

Reply:
I have now added some additional typing on the typescript side

```typescript
export type GlobalDataForAnalysisPy = {
  sampling?: {
    draws: number[][];
    paramNames: string[];
    numChains: number;
    chainIds: number[];
  };
};
```